app/src/main/java/app/revanced/integrations/shared/GmsCoreSupport.java

```diff
@@ -73,6 +74,21 @@ private static void showBatteryOptimizationDialog(Activity context,
     @RequiresApi(api = Build.VERSION_CODES.N)
     public static void checkGmsCore(Activity context) {
         try {
+            // Verify the user has not included GmsCore for a root installation.
+            // GmsCore Support changes the package name, but with a mounted installation
+            // all manifest changes are ignored and the original package name is used.
+            if (context.getPackageName().equals(ORIGINAL_UNPATCHED_PACKAGE_NAME)) {
+                Logger.printInfo(() -> "App is mounted with root, but GmsCore patch was included");
+                // Cannot use localize text here, since the app will load
+                // resources from the unpatched app and all patch strings are missing.
+                Utils.showToastLong("'GmsCore support' patch cannot be used with root mount installation");
```

Comment:
Toasts tend to be limited in length, a shorter suggestion would be:

```suggestion
                Utils.showToastLong("The 'GmsCore support' patch breaks mount installations");
```